utils/kata-manager.sh

```diff
@@ -244,6 +247,7 @@ Options:
                 Find more details on LTS and Active versions of containerd on
                 https://containerd.io/releases/#support-horizon
  -d           : Enable debug for all components.
+ -D           : Install Docker CLI.
```

Comment:
This might be more informative?

```suggestion
 -D           : Install Docker server and CLI tooling (takes priority over '-c').
```